sources/packages/web/src/components/aest/StudentApplications.vue

```diff
@@ -80,7 +85,12 @@
                 "
               />
             </v-btn>
-            <v-btn :disabled="hasRestriction" plain>
+            <v-btn
+              :disabled="
+                hasRestriction || sinValidStatus !== SINStatusEnum.VALID
```

Comment:
Is this validation happening also in the API?

Reply:
Yes validation is happening in the API. I created a decorator called **@CheckSinValidation()** in the previous PR I worked on.

Reply:
Nice work with the decorator, please just include it in _auth.module.ts_ as we talked 😉